src/SimpleThings/EntityAudit/Collection/AuditedCollection.php

```diff
@@ -1,10 +1,10 @@
 <?php
+
 /*
  * (c) 2011 SimpleThings GmbH
  *
  * @package SimpleThings\EntityAudit
  * @author Benjamin Eberlei <eberlei@simplethings.de>
- * @author Andrew Tch <andrew.tchircoff@gmail.com>
```

Comment:
This is wrong

if we use the header, please move the author above the class name and check the other files